src/templates/Templates.js

```diff
@@ -450,18 +455,21 @@ const StateTemplates = {
   Procedure: {
     type: "Procedure",
     codes: [{...TypeTemplates.Code.Snomed}],
-    duration: {
-      low: 30,
-      high: 30,
-      unit: 'minutes'
-    }
+    distribution: {
+      "kind": "UNIFORM",
+      "parameters": {
+        "high": 60,
+        "low": 30
+      }
+    },
+    "unit": "minutes"
   },
 
   VitalSign: {
     type: "VitalSign",
     vital_sign: "",
     unit: "",
-    exact: {...AttributeTemplates.Exact}
+    distribution: {kind: "EXACT", parameters: {value: 1}}
   },
 
   Observation: {
```

Comment:
Should there be a change to the Observation state as well?

Reply:
Yes. This has now been fixed.